build/automation/var/platform-texas/v1/account-live-k8s-nonprod.mk

```diff
@@ -1,5 +1,8 @@
 AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_NONPROD)
 AWS_ACCOUNT_NAME := nonprod
+AWS_CERTIFICATE = c0718115-4e22-4f48-a4aa-8c16ea86c5e6
```

Comment:
Is this DoS CS API domain-specific certificate? It may need to be moved to project.mk - otherwise (if this is *.k8s-nonprod.texasplatform.uk) then that's all fine leaving it in here.

Reply:
Providing the above is correct, can we have a PR to be created for that change to the Make DevOps repository as this is the library file? Thanks

Reply:
Its the texas cert.  Will update the name to include _ID